gradle.properties

```diff
@@ -1,42 +1,55 @@
-# IntelliJ Platform Artifacts Repositories
-# -> https://www.jetbrains.org/intellij/sdk/docs/reference_guide/intellij_artifacts.html
-
 pluginGroup = com.chriscarini.jetbrains
-pluginName = 'Git Push Reminder'
-# SemVer format -> https://semver.org
-pluginVersion = 2.0.5
+pluginName = Git Push Reminder
+pluginRepositoryUrl = https://github.com/ChrisCarini/git-push-reminder-jetbrains-plugin
+# Plugin Version in SemVer format -> https://semver.org
+# To release a new version of the plugin against an EAP build of IntelliJ,
+# the pluginVersion should be suffixed with '-EAP'. This will release the
+# plugin to the below JetBrains-hosted plugin repository:
+#   - https://plugins.jetbrains.com/plugins/eap/list
+# Note: You will need to configure the above URL as a custom plugin repository;
+#       see directions: https://www.jetbrains.com/help/idea/managing-plugins.html#repos
+pluginVersion = 2.1.0
 
 ## See https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html
 ## for insight into build numbers and IntelliJ Platform versions.
-pluginSinceBuild = 241
-pluginUntilBuild = 241.*
+pluginSinceBuild = 242
+pluginUntilBuild = 242.*
 
 # Plugin Verifier integration -> https://github.com/JetBrains/gradle-intellij-plugin#plugin-verifier-dsl
 # See https://jb.gg/intellij-platform-builds-list for available build versions
-pluginVerifierIdeVersions = 2024.1.5,LATEST-EAP-SNAPSHOT
+pluginVerifierIdeVersions = 2024.2,LATEST-EAP-SNAPSHOT
 # Failure Levels: https://github.com/JetBrains/gradle-intellij-plugin/blob/master/src/main/kotlin/org/jetbrains/intellij/tasks/RunPluginVerifierTask.kt
-# Exclude `NOT_DYNAMIC` Failure Level because we make use of `projectCloseHandler` (in `plugin.xml`)
-# which is considered to be a non-dynamic feature.
 pluginVerifierExcludeFailureLevels = NOT_DYNAMIC
```

Comment:
```suggestion
# Exclude `NOT_DYNAMIC` Failure Level because we make use of `projectCloseHandler` (in `plugin.xml`)
# which is considered to be a non-dynamic feature.
pluginVerifierExcludeFailureLevels = NOT_DYNAMIC
```